frontend/src/lib/components/ChartFilter/chartFilterLogic.ts

```diff
@@ -20,8 +23,21 @@ export const chartFilterLogic = kea<chartFilterLogicType>({
                 setChartFilter: (_, { filter }) => filter,
             },
         ],
+        highlightChartChange: [
+            false,
+            {
+                chartAutomaticallyChanged: () => true,
+                endHighlightChange: () => false,
+            },
+        ],
+        initialLoad: [
+            true,
+            {
+                setInitialLoad: () => false,
```

Comment:
Normally actions `setX(x)` let you set a reducer to whatever variable you give the action. In this case "set initial load" sets "initial load" to "false", which is unintuitive. The easiest fix is calling the action something like `finishedInitialLoad`.